getting_started/topics/first-boot/distribution-files-product.adoc

```diff
@@ -3,14 +3,16 @@
 
 The {project_name} Server is contained in one distribution file:
 
-* 'RH-SSO-{project_version}.[zip|tar.gz]'
-
-The 'RH-SSO-{project_version}.[zip|tar.gz]' file is the server-only distribution. It contains only the scripts and binaries
-to run the {project_name} server.
-
-To unpack these files run the `unzip` or `gunzip` and `tar` utilities.
-
+* 'rh-sso-{project_version}.GA.zip'
 
+NOTE: This file can be downloaded in the Red Hat customer portal
+https://access.redhat.com/downloads[Downloads] section, within the "JBoss
+Integration and Automation" subsection.
 
+.Download Page
+image:{project_images}/download-page.png[]
```

Comment:
We shouldn't include a screenshot of this. The list of downloads will change from release to release and we can't keep that updated.

Reply:
Fixed